### ISSUE TITLE
Confusion about comments in raft.rs at line 1630

### ISSUE THREAD

**Issue by wego1236** [tikv/raft-rs#527]
https://github.com/tikv/raft-rs/blob/cd621e4c7bdf4278e970e13ea236bb0559c9f21b/src/raft.rs#L1629C45-L1630C60

I was reviewing the code in src/raft.rs and got a bit confused about the comments at line 1630. 
It has previously stated that the content of the letter replied to from follower is 'RejectHint of 6 and LogTerm of 2', but here it says that it tells the leader that it is still at term 2 at index 9. I think this should be changed to ' But the follower already told the leader that it is still at term 2 at index 6' .
In other words, i think this should be changed from index 9 to 6

**Comment by BusyJay** [comment]
You are correct. Can you send a PR to fix it?

**Comment by wego1236** [comment]
Sure, I'll go now.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年9月14日(星期四) 下午2:40
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [tikv/raft-rs] Confusion about comments in raft.rs at line 1630 (Issue #527)





 
You are correct. Can you send a PR to fix it?
